doc/sphinx/reference/varnishncsa.rst

```diff
@@ -34,6 +34,30 @@ The following options are available:
 
 .. include:: ../include/varnishncsa_options.rst
 
+BACKEND MODE
```

Comment:
Backend mode being a title of it own suggests that backend_mode is something added onto rather than being a first class citizen. I'd prefer a title of 'MODES' that discusses the different modes.